src/styles/block/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+
+  &__img {
+    margin-left: 3px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 36px;
+    margin-bottom: 4px;
+    font-weight: $font-weight-medium;
+    font-size: $font-size-normal;
+    line-height: 18px;
+    color: $color-dark;
+  }
+  
+  &__code {
+    font-size: $font-size-small;
+    margin: 0;
+    font-weight: $font-weight-normal;
+    line-height: 14px;
+    color: $color-grey;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+  
+  &__rating-value {
+    font-size: $font-size-small;
+    line-height: 14px;
+    color: $color-dark;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  &__price-text {
+    margin-top: 25px;
+    font-weight: $font-weight-normal;
+    font-size: $font-size-normal;
+    color: $color-grey;
+  }
+
+  &__price-cost {
+    margin-top: 22px;
+    font-weight: $font-weight-bold;
+    font-size: $font-size-big;
+    line-height: 18px;
+    color: $color-dark;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: $color-blue;
+    border: $color-blue;
+    border-radius: 5px;
+    height: 40px;
+    box-sizing: border-box;
+    line-height: 40px;
```

Comment:
Button has a display: flex and align-items: center, you don't really need line-height for vertical alignment
```suggestion
```